src/interfaces/IKycProvidersManager.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: CC0-1.0
+
+pragma solidity 0.8.15;
+
+interface IKycProvidersManager {
+    function passesKYC(address _addr) external view returns (bool);
+    function getAssociatedLimitsTier(address _addr, bool _checkKYC) external view returns (uint8);
```

Comment:
Why do we need 2 methods here? Can we use a single method, returning tuple of `bool` and `uint8` instead?

Reply:
I would keep the method `passesKYC` for the cases when it could be required to get KYC status without necessity to get the tier.

`getAssociatedLimitsTier` has been replaced by `getIfKYCpassedAndTier` to combine two actions in one call: https://github.com/zkBob/zkbob-contracts/pull/54/commits/62a721e883ebe38e6cb3f1c3cc82c9738f3c543e